src/ci-line-chart.js

```diff
@@ -277,14 +285,14 @@ const vis = {
         }
       });
       cfg.data.datasets.push({
-        label: `${optionsToFriendly[config.field_y]}-${group}`,
+        label: `${optionsToFriendly[config.field_y]}` + groupLabel,
```

Comment:
```suggestion
        label: `${optionsToFriendly[config.field_y]}${groupLabel}`,
```
Suggestion to include the groupLabel in the existing template